extension/src/validator/error-messages.js

```diff
@@ -15,5 +15,6 @@ module.exports = {
   GET_PAYMENT_METHODS_REQUEST_INVALID_JSON: 'getOriginKeysRequest does not contain valid JSON.',
   MAKE_PAYMENT_REQUEST_INVALID_JSON: 'makePaymentRequest does not contain valid JSON.',
   // eslint-disable-next-line max-len
-  SUBMIT_ADDITIONAL_PAYMENT_DETAILS_REQUEST_INVALID_JSON: 'submitAdditionalPaymentDetailsRequest does not contain valid JSON.'
+  SUBMIT_ADDITIONAL_PAYMENT_DETAILS_REQUEST_INVALID_JSON: 'submitAdditionalPaymentDetailsRequest does not contain valid JSON.',
+  AMOUNT_PLANNED_INVALID: 'amountPlanned field cannot be changed after makePayment request is made to Adyen'
```

Comment:
it's valid but immutable, maybe change the naming in here to 
"AMOUNT_PLANNED_MODIFICATION_IS_NOT_ALLOWED"

Also, the message here is for the user, could we say more precisely: 

`Modifying the "amountPlanned" value is not allowed after the payment amount charged by Adyen.`

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/283/commits/fd30c79d4afcf8cbb87739bb7a3d268700fc9e51

I will not change the message to your message as it's probably not true - amountPlanned is not charged in some cases, etc. you need to do the redirect and then verification etc. 
And the real reason for this error is simply after there's a makePayment request, amountPlanned cannot be changed.